mismi-sqs/src/Mismi/SQS/Commands.hs

```diff
@@ -44,11 +44,21 @@ createQueue q v = do
     (pure . QueueUrl)
     (res ^. cqrsQueueURL)
   where
-    -- If queue alsready exists (and has different VisibilityTimeout)
+    -- If queue already exists (and has different VisibilityTimeout)
     handleExists = handling _QueueNameExists $ \_ ->
       -- Get existing queue (using default parameters)
       send $ A.createQueue (renderQueueName q)
 
+
+getQueueByName :: QueueName -> Maybe Int -> AWS QueueUrl
```

Comment:
One downside here is that in the happy path we are doing an extra api call

Reply:
Yeah, its a matter of what is most common, getting an existing queue or creating a new one. I suspect its the former.